src/client.ts

```diff
@@ -332,22 +332,32 @@ export class StreamChat<
   _hasConnectionID = () => Boolean(this.connectionID);
 
   /**
-   * setUser - Set the current user, this triggers a connection to the API
+   * connectUser - Set the current user, this triggers a connection to the API
```

Comment:
```suggestion
   * connectUser - Set the current user and open a WebSocket connection
```

same thing for `connectAnonymousUser` to clarify a bit further?